docs/reference/sections/dom-binding.md

```diff
@@ -1,6 +1,21 @@
-Full documentation for the DOM Binding module is coming soon.
+DOM Binding is powered by Rivets.js, providing a flexible templating approach to cart rendering.
 
-In the meantime, you can check out the example on the [front page][] in combination with the [Rivets.js documentation][] to get an idea of how things work.
+See the [DOM Binding] section in the Guide for more.
 
-[front page]: /
-[Rivets.js documentation]: http://rivetsjs.com/
+### Rivets Binders
+
+A full breakdown of Rivets' available binders can be found under the Rivets docs [Binder Reference][].
+
+### Formatters
+The general syntax for formatters looks similar to Liquid filters, with a couple of key differences. In generic form, it can be seen as 
+```
+property | formatter
+```
+
+with the optional addition of arguments that can be passed to the formatter. In contrast to Liquid filters, formatters with arguments are not separated by colons, e.g. 
```

Comment:
Ok - I didn't call this out as it works the same as Shopify filters, but I know this has been a confusion point due to Rivets changing it's syntax - happy to add in though to ensure it's clear.